packages/plugin-session-replay-browser/src/session-replay.ts

```diff
@@ -15,7 +45,12 @@ export class SessionReplayPlugin implements EnrichmentPlugin {
     this.options = { ...options };
   }
 
-  async setup(config: BrowserConfig) {
+  async setup(config: BrowserConfig, client?: BrowserClient) {
+    if (!client) {
+      config.loggerProvider.error('SessionReplayPlugin requires v1.9.1+ of the Amplitude SDK.');
+      return;
+    }
```

Comment:
In our docs/guides we already recommend v1.9.1+

Reply:
good to know we already are! there might be some spots that need updating like: https://github.com/amplitude/Amplitude-TypeScript/blob/v1.x/packages/plugin-session-replay-browser/README.md?plain=1#L26

Reply:
Updated README